ansible/events.yml

```diff
@@ -0,0 +1,26 @@
+- hosts: events
+  roles:
+    - common
+    - webserver
+    - events
+  vars:
+    docker_dir: "/data/docker"
+
+- hosts: elasticsearch
```

Comment:
As other services uses elasticsearch too, maybe this group should be renamed to events-elasticsearch to avoid future conflicts

Reply:
better `events_elasticsearch` following https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_variables.html#creating-valid-variable-names